modules/about-must-gather.adoc

```diff
@@ -41,7 +41,8 @@ $ oc adm must-gather -- /usr/bin/gather_audit_logs
 +
 [NOTE]
 ====
-Audit logs are not collected as part of the default set of information to reduce the size of the files.
+. Audit logs are not collected as part of the default set of information to reduce the size of the files.
+. In Windows, the cwRsync client should be installed and added to the PATH for use with the oc rsync command.
```

Comment:
See IBM Style for [Windows](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#windows).
Consider rewording to avoid passive voice, for example:
```suggestion
* On a Windows operating system, install the `cwRsync` client and add to the `PATH`  variable for use with the `oc rsync` command.
```